db/migrations/20191112145233_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = (knex) => {
+  return knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return knex.raw('ALTER TABLE movies ADD CONSTRAINT movies_name_not_null CHECK (name IS NOT NULL) NOT VALID')
+    .then(() => {
```

Comment:
Looks like this is a nested `then` block which I would avoid doing. You can add the `then` block to the outer function ending in line 11 or even better I'd suggest using `async/await`.